ps_mbo.php

```diff
@@ -207,4 +207,22 @@ private function getTraitNames(): array
 
         return $traits;
     }
+
+    /**
+     * Used to correctly check if the module is enabled or not whe registering services
+     *
+     * @return bool
+     */
+    public static function checkModuleStatus(): bool
+    {
+        $result = Db::getInstance()->getRow('SELECT m.`id_module` as `active`, ms.`id_module` as `shop_active`
```

Comment:
Point of attention about this query when we are in multishop. `getRow` automatically adds LIMIT 1 to the query. We need to be sure when MBO is activated/disabled, it will be in the good status for all the shops

Reply:
I will add another PR to ensure the fact that MBO is enabled / disabled in all shops on a multishop context.